packages/fetch-engine/src/getLatestTag.ts

```diff
@@ -56,6 +56,8 @@ export function getAllUrls(branch: string, commit: string): string[] {
     'freebsd',
     'arm',
     'linux-nixos',
+    'linux-arm-openssl-1.1.x',
+    'linux-arm-openssl-1.0.x',
```

Comment:
This is the old name - why does this have to be listed here now?
(might be worth a comment)

Reply:
This is the exclude list, platforms we know exists but doesn't compile for like `freebsd` and `linux-nixos`. We don't compile for 32bit arm so we need to list it here so that we don't try to download it.

Reply:
Yeah let's make that more obvious in the code with a comment above the definition - I like the words you used already.

Reply:
[`86cd99a` (#173)](https://github.com/prisma/engines-wrapper/pull/173/commits/86cd99a2d6d0c3f96e62b8e7d0c4373e945560f7)